### PR TITLE
MINOR: parameterize group-id in GroupMetadataManagerTestContext

### PR DESCRIPTION
**Pull request by dongnuo123**
This pr parameterize some group ids in GroupMetadataManagerTestContext that are now constant strings.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTestContext.java [hunk 1]
```diff
@@ -901,7 +901,7 @@ public RebalanceResult staticMembersJoinAndRebalance(
     public PendingMemberGroupResult setupGroupWithPendingMember(ClassicGroup group) throws Exception {
         // Add the first member
         JoinGroupRequestData joinRequest = new JoinGroupRequestBuilder()
-            .withGroupId("group-id")
+            .withGroupId(group.groupId())
```
Comment (chia7712) [inline group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTestContext.java:904]: (this is unrelated to this PR)

The caller `GroupMetadataManagerTest` has similar issue (https://github.com/apache/kafka/blob/trunk/group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java#L7631). Should it get fixed also?

**Review by chia7712 — APPROVED** [review]
LGTM

a trivial comment left

File: group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTestContext.java (hunk 1 above)
Comment (dongnuo123) [reply group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTestContext.java:904]: The group id is not a parameter of the method, so I guess it's fine to use `"group-id"`
